docs/api/animation/AnimationAction.html

```diff
@@ -11,121 +11,363 @@
     <h1>[name]</h1>
 
     <div class="desc">
-    An AnimationAction schedules clip playback on specific objects.
+        AnimationActions schedule the performance of the animations which are stored in
+        [page:AnimationClip AnimationClips].<br /><br />
+        
+        Note: Most of AnimationAction's methods can be chained.<br /><br />
+        
+        For an overview over the different elements of the three.js animation system see the
+        "Animation System" article in the "Next Steps" section of the manual.
     </div>
-
+    
 
     <h2>Constructor</h2>
-
+    
 
     <h3>[name]( [page:AnimationMixer mixer], [page:AnimationClip clip], [page:Object3D localRoot] )</h3>
-
-
+    <div>       
+        [page:AnimationMixer mixer] - the *AnimationMixer* that is controlled by this action.<br />
+        [page:AnimationClip clip] - the *AnimationClip* that holds the animation data for this action.<br />
+        [page:Object3D localRoot] - the root object on which this action is performed.<br /><br />
+        
+        Note: Instead of calling this constructor directly you can instantiate an AnimationAction with 
+        [page:AnimationMixer.clipAction].
+    </div>   
+    
+   
     <h2>Properties</h2>
+       
+        
+    <h3>[property:Boolean clampWhenFinished]</h3>
+    <div>
+        If *clampWhenFinished* is set to true, [page:.paused paused] will automatically be switched to 
+        true, when the last loop of the action has finished. Therefore the last frame of the animation 
+        will be hold as the final state.<br /><br />
+         
+        If *clampWhenFinished* is set to false, [page:.enabled enabled] will automatically be switched 
+        to false, when the last loop of the action has finished, so that this action has no further 
+        impact.<br /><br />
+        
+        Default is false.<br /><br />
+        
+        Note: *clampWhenFinished* has no impact, if the action is interrupted (it has only an effect if 
```

Comment:
>Note: *clampWhenFinished* has no impact, if the action is interrupted... 

Another unnecessary comma, change to

> >Note: *clampWhenFinished* has no impact if the action is interrupted...